.github/pull_request_template.md

```diff
@@ -51,12 +42,10 @@ This PR has:
 - [ ] added integration tests.
```

Comment:
Since you're updating the template, line 37 (2nd item in PR checklist) has an extra indent that can be removed

Reply:
If you're referring to the 'concurrency checklist' thing, i believe it's supposed to be a sub-item of 'self reviewed'

Reply:
That said, I'm open to changing it, either way seems fine to me

Reply:
Ah, if it's a sub item of "self reviewed", then it's fine as is.